checkbox-provider/lib/qat_tools-1.0.0/setup.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+from setuptools import setup
+
+setup(
+    name="qat_tools",
+    url="https://github.com/canonical/qat-tools",
```

Comment:
Is this a private repo? I don't seem to have access. And from the `debian` folder it seems we plan to distribute this as a .deb as well?

The `qat_tools` work is amazing, so I was wondering whether it warranted putting in its own repo? I'm also wondering about maintainability and whether we may be duplicating work that Intel has already done. I haven't found a centralized QAT lib in python from Intel but I do stumble across random scripts here and there.

Reply:
i did read intel scripts and got inspired for several things, for testing, we cannot take intel scripts as they are because they are not suitable to be included in tests, i don't think we duplicate work here